run.py

```diff
@@ -1,19 +1,32 @@
-from matplotlib import pyplot as plt
-import numpy as np
-import pymysql
+from contextlib import contextmanager
+import logging
 import time
-import coloredlogs, logging
 
-def run(query, host='localhost', user='root', passwd='', log=True):
-    conn = pymysql.connect(host=host, port=3306, user=user, passwd=passwd, db='mysql')
-    start = time.time()
+import coloredlogs
+from matplotlib import pyplot
+import numpy
+import pymysql
+
+logger = logging.getLogger("run")
 
+@contextmanager
+def no_logs():
+    level = logger.level
+    logger.setLevel(logging.WARN)
+    try:
+        yield None
+    finally:
+        logger.setLevel(level)
+
+def run(query, host="localhost", user="root", passwd=""):
+    conn = pymysql.connect(host=host, port=3306, user=user, passwd=passwd, db="mysql")
```

Comment:
`port=3306` & `db="mysql"` could be a default arguments to function `run`

Reply:
TODO in the next PR 👍